tests/patmat/t9232.check

```diff
@@ -1,5 +1,5 @@
 ./tests/patmat/t9232.scala:13: warning: match may not be exhaustive.
-It would fail on the following input: Node1(_), Node2()
+It would fail on the following input: Node2()
```

Comment:
Yes, it's imprecise. Currently I take **extractor** to match everything. I'll need to refine the algorithm for a better modelling of **extractors**.